NetworkChatServer/src/chat/handler/ClientHandler.java

```diff
@@ -80,8 +81,13 @@ private void readMessages() throws IOException {
             System.out.println("message: " + message);
             if (message.startsWith(END_CMD)) {
                 return;
-            }
-            else {
+            } else if (message.startsWith(PRIVATE_MSG_CMD)) {
+                String[] parts = message.split(" ");
```

Comment:
```suggestion
                String[] parts = message.split(" ", 3);
```
если сообщение содержит пробелы, то мы его поделим также на части в массив. Со вторым параметром нам достаточно будет взять `parts[2]`